modules/benchmark/benchmark_runner.py

```diff
@@ -118,14 +140,14 @@ def __init__(self,
         self.exceptions_caught = []
         self.log_eval_avg_every = log_eval_avg_every
         self.logger = logging.getLogger(logger_name or "BenchmarkRunner")
-        self.logger.setLevel(logging.DEBUG)
+        self.logger.setLevel(logging.INFO)
 
     def _create_configurations(self, num_scenarios=None):
         benchmark_configs = []
-        for behavior_name, behavior_bark in self.behaviors.items():
-            # run over all scenario generators from benchmark database
-            for scenario_generator, scenario_set_name in self.benchmark_database:
-                for scenario, scenario_idx in scenario_generator:
+        # run over all scenario generators from benchmark database
+        for scenario_generator, scenario_set_name in self.benchmark_database:
+            for scenario, scenario_idx in scenario_generator:
+                for behavior_name, behavior_bark in self.behaviors.items():
```

Comment:
is there any reason you changed the order of the loops?

Reply:
The previous order of the loops generated a sequence of configurations so that all scenarios would be run for on behavior before any other behavior is tested.

1) Scenario A, Behavior A
2) Scenario B, Behavior A
3) Scenario A, Behavior B
4) Scenario B, Behavior B

 When saving intermediate results, it appeared more useful first to complete a scenario with all behaviors, than running all scenarios on the same behavior. The intermediate result should then still be meaningful even though not all scenarios have been tested yet, compared to the previous order, where we would only have the results for one single behavior.

1) Scenario A, Behavior A
2) Scenario A, Behavior B
3) Scenario B, Behavior A
4) Scenario B, Behavior B

Reply:
I guess the order does not really matter for the pandas table. Thus, should be okay.